pkg/config/containers.conf

```diff
@@ -61,8 +67,8 @@
 
 # List of default sysctls. If it is empty or commented out, only the sysctls
 # defined in the container json file by the user/kube will be added.
-# default_sysctls = [
-# ]
+#
+# default_sysctls = []
```

Comment:
Need to know what the format for this is

Reply:
Fixed